src/main/java/com/sonyericsson/jenkins/plugins/bfa/db/KnowledgeBase.java

```diff
@@ -50,7 +50,9 @@
  * @author Robert Sandell &lt;robert.sandell@sonyericsson.com&gt;
  */
 public abstract class KnowledgeBase implements Describable<KnowledgeBase>, Serializable {
-
+	
+	private static final long DEFAULT_FAILURE_CAUSES = 0;
```

Comment:
DEFAULT_**NO_OF_**FAILURE_CAUSES or similar to not confuse with any potential list of default failure causes.